pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "neobolt==1.7.17",
     "six==1.16.0",
 ]
-dynamic = ["version"]
+version='4.0.10'
```

Comment:
This will change (and potentially break) the Publish Package pipeline behavior. It's OK, just need to be aware of it, or find a way around it.